package.json

```diff
@@ -28,8 +28,8 @@
   },
   "dependencies": {
     "bluebird": "3.4.6",
-    "body-parser": "1.18.2",
-    "chai": "^3.4.1",
+    "body-parser": "1.17.2",
```

Comment:
Some of these package changes don't look right. They look like downgrades. Also, edit out the `false` prefixes below, I think it is a bug with yarn.

Reply:
@jsachs I believe downgrades to body-parser were from @RuchitaGarde's branch which this PR is based off of. But I'm not sure if there was a reason for the downgrade. Ruchita did you have any reason for downgrading?

Reply:
@RuchitaGarde